src/devices/heiman.ts

```diff
@@ -1176,4 +1177,105 @@ export const definitions: DefinitionWithExtend[] = [
         },
         endpoint: (device) => ({default: 1}),
     },
+    {
+        fingerprint: [{modelID: "HS2AQ-EF-3.0", manufacturerName: "HEIMAN"}],
+        model: "HS2AQ-EF-3.0",
+        vendor: "Heiman",
+        description: "Air quality monitor",
+        extend: [
+            addCustomClusterHeimanSpecificAirQualityShort(),
+
```

Comment:
```suggestion
m.battery(), m.humidity(), 
```
Use this, by using this you can remove the code in fromZigbee, configure and exposes